repo/packages/S/spark/13/config.json

```diff
@@ -0,0 +1,235 @@
+                            "default": "",
+                            "type": "string"
+                        }
+                    }
+                }
+            }
+        },
+        "security": {
+            "description": "Spark security configuration properties",
+            "type": "object",
+            "properties": {
+                "kerberos": {
+                    "description": "Spark Kerberos configuration.",
+                    "type": "object",
+                    "properties": {
+                        "krb5conf": {
+                            "description": "Base64 encoded krb5.conf file.  Copied to the drivers and executors so that they may access your KDC.  Also used by the history server.",
+                            "type": "string",
+                            "default": ""
+                        }
+                    }
+                },
+                "mesos": {
+                    "description": "Mesos scheduler configuration properties.",
+                    "type": "object",
+                    "properties": {
+                        "ssl": {
+                            "description": "Mesos scheduler SSL configuration.",
+                            "type": "object",
+                            "properties": {
+                                "enabled": {
+                                    "default": false,
+                                    "type": "boolean"
+                                },
+                                "cert_file": {
+                                    "type": "string",
+                                    "description": "Absolute path to scheduler's cert.  This is required if security.mesos-ssl.enabled is true."
+                                },
+                                "key_file": {
+                                    "type": "string",
+                                    "description": "Absolute path to scheduler's private key.  This is required if security.mesos-ssl.enabled is true."
+                                },
+                                "CA_dir": {
+                                    "type": "string",
+                                    "description": "Absolute path to a directory containing CA certs."
+                                }
+                            }
+                        },
+                        "authentication": {
+                            "description": "Mesos scheduler dcos-oauth configuration.",
+                            "type": "object",
+                            "properties": {
+                                "enabled": {
+                                    "description": "Whether to enable dcos-oauth.",
+                                    "type": "boolean",
+                                    "default": false
+                                },
```

Comment:
Instead of checking for the "enabled" boolean, you can just check for the presence of the "secret"